addons/test_portal/controllers.py

```diff
@@ -0,0 +1,8 @@
+from odoo import http
+from odoo.addons.portal.controllers.portal import CustomerPortal
+
+
+class Portal(CustomerPortal):
+    @http.route(['/my/thing', '/my/thing2'], type='http', auth="user", website=True)
+    def thing(self, **_):
```

Comment:
to not conflict with translation if we use it one day here
```suggestion
    def thing(self, **__):
```